contrib/go/src/python/pants/contrib/go/targets/go_target.py

```diff
@@ -65,3 +65,10 @@ def import_path(self):
     :returns: An import path that can be used to import this package in a `.go` file.
     :rtype: string
     """
+
+  def __init__(self, build_flags=None, **kwargs):
```

Comment:
It would be nice to keep `__init__` above other instance methods in the file since we do this ~everywhere else I know of.